server/routes/incidentDetails/incidentDetailsValidation.ts

```diff
@@ -75,9 +75,13 @@ const errors: { [key: string]: FormError } = {
     href: '#discoveryDate[date]',
     text: 'The incident discovery time must be in the past',
   },
-  DISCOVERY_AFTER_INCIDENT_TIME: {
+  DISCOVERY_AFTER_INCIDENT_DATE: {
     href: '#discoveryDate[date]',
-    text: 'The incident discovery date/time must be after the indident date/time',
+    text: 'The discovery date must be after the indident date',
+  },
+  DISCOVERY_AFTER_INCIDENT_TIME: {
+    href: '#discoveryDate[time][minute]',
+    text: 'The discovery time must be after the indident time',
```

Comment:
This is where the spelling error is being produced

Reply:
fixed